trajopt/include/trajopt/problem_description.hpp

```diff
@@ -85,6 +85,7 @@ struct TRAJOPT_API TrajOptResult
   std::vector<std::string> cost_names, cnt_names;
   DblVec cost_vals, cnt_viols;
   TrajArray traj;
+  sco::OptStatus status;
```

Comment:
Since these are populated from the constructor should we make everything const in this struct?

Reply:
I think it could, but I believe it's slightly outside the scope of that PR. I gave maintainer access anyways, feel free to add what you deem necessary, but I aimed at keeping it simple.